src/main/java/org/folio/orders/events/handlers/MessageAddress.java

```diff
@@ -2,7 +2,8 @@
 
 public enum MessageAddress {
 
-  ORDER_STATUS("org.folio.orders.order.update.status");
+  ORDER_STATUS("org.folio.orders.order.update.status"),
+  RECEIPT_STATUS("org.folio.orders.receipt.status");
```

Comment:
I'd follow the same general naming convention... maybe something like:

`org.folio.orders.order-line.update.receiptStatus` or 
`org.folio.orders.order-line.update.receipt-status` or
`org.folio.orders.po-line.update.receiptStatus` or 
`org.folio.orders.po-line.update.receipt-status`

Reply:
Using `org.folio.orders.po-line.update.receipt-status`